app/models/manager_refresh/save_collection/saver/sql_helper.rb

```diff
@@ -36,8 +41,13 @@ def build_insert_query(inventory_collection, all_attribute_keys, hashes)
         insert_query
       end
 
+      def build_update_set_cols(key)
+        "#{ActiveRecord::Base.connection.quote_column_name(key)} = updated_values.#{ActiveRecord::Base.connection.quote_column_name(key)}"
+      end
+
       def build_update_query(inventory_collection, all_attribute_keys, hashes)
         all_attribute_keys_array = all_attribute_keys.to_a
+        all_attribute_keys_array.delete(:type)
```

Comment:
removed, I will fix it in follow up. I think I will just allow updating of the type though